src/Avalonia.DesignerSupport/Remote/RemoteDesignerEntryPoint.cs

```diff
@@ -169,6 +175,35 @@ class AppInitializer : IAppInitializer
         public static void Main(string[] cmdline)
         {
             var args = ParseCommandLineArgs(cmdline);
+            if (args.Debug)
+            {
+                // According this https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.debugger.launch?view=net-6.0#remarks
+                // documentation, on not windows platform Debugger.Launch() always return true without running a debugger.
+                if (System.Diagnostics.Debugger.Launch())
```

Comment:
I feel we have exact same logic somewhere else. Maybe move it to a shared cs file?